packages/spindle-ui/src/Form/DropDown.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback, useRef, useState } from 'react';
+
+import { ChevronDownBold } from '../Icon';
+
+type Props = {
+  hasError?: boolean;
+} & React.SelectHTMLAttributes<HTMLSelectElement>;
+
+const BLOCK_NAME = 'spui-DropDown';
+
+export const DropDown: React.FC<Props> = ({
+  children,
+  hasError = false,
+  onChange,
+  placeholder = '',
+  ...rest
+}: Props) => {
+  const selectEl = useRef<HTMLSelectElement>(null);
+  const [value, setValue] = useState('');
+  const handleChange = useCallback(
+    (event) => {
+      if (typeof onChange === 'function') {
+        onChange(event);
+      }
+      if (selectEl && selectEl.current) {
+        const optionEl = selectEl.current.querySelector(
+          `option[value="${selectEl.current.value}"]`,
+        ) as HTMLElement;
+        const newValue = optionEl.innerText;
+        setValue(newValue);
+      }
+    },
+    [value],
+  );
+  return (
+    <label
+      className={[
+        `${BLOCK_NAME}-label`,
+        value && !hasError ? 'is-active' : '',
+        hasError ? 'is-error' : '',
+      ]
+        .filter(Boolean)
+        .join(' ')}
+    >
+      <span className={`${BLOCK_NAME}-visual`}>{value || placeholder}</span>
+      <span className={`${BLOCK_NAME}-icon`}>
+        <ChevronDownBold aria-label="Chevron down" />
+      </span>
+      <select
+        className={`${BLOCK_NAME}-select`}
+        ref={selectEl}
+        onChange={handleChange}
+        {...rest}
+      >
+        {children}
+      </select>
+      <span className={`${BLOCK_NAME}-outline`}></span>
```

Comment:
[nits] 好みの問題だと思うのですが…w（僕のIDEだと`XML tag has empty body`）
```suggestion
      <span className={`${BLOCK_NAME}-outline`} />
```

Reply:
でもlintが何も言わない設定なのでこのままで良さそうですね:pray:

Reply:
そだねー、ここはほんとにどちらでもいいんだけど、いちおlint設定ルールでもいいかなと思いつつ、IDEは[Intellijか何か](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360005021160-XML-tag-has-empty-body)かな!?

https://github.com/openameba/spindle/blob/b7d267e3a93700ee1f1e4e959fbd5c396af7d559/.eslintrc.json#L6

Reply:
> IDEはIntellijか何かかな!?

その通りですwそうですね！lint設定のままで:pray: